piker/brokers/kraken.py

```diff
@@ -275,14 +443,396 @@ async def bars(
 async def get_client() -> Client:
     client = Client()
 
+    ## TODO: maybe add conditional based on section
+    section = get_config()
+    client._name = section['key_descr']
+    client._api_key = section['api_key']
+    client._secret = section['secret']
+    ## TODO: Add a client attribute to hold this info
+    #data = {
+    #    # add non-nonce and non-ofs vars
+    #}
+
     # at startup, load all symbols locally for fast search
     await client.cache_symbols()
 
     yield client
 
 
-async def stream_messages(ws):
+def pack_position(
```

Comment:
Not totally sure i follow but it seem mostly benign for now.

Reply:
As mentioned in chat stick all the pp calcing stuff in this routine and keep the method `get_positions()` to be something like `get_trades()` and just have it pull from the API. This will also make is easier to swap in similar `cryptofeed` apis later if need be.